src/ir/auth_logic/ast_equality_test.cc

```diff
@@ -0,0 +1,64 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
```

Comment:
Copyright year.

This should also go into a file predicate_test.cc

Reply:
fixed